Makefile

```diff
@@ -87,6 +87,10 @@ test-e2e: ## Run openshift specific e2e test
 	# After success, merge all JUnit files into one
 	hack/junitmerge.sh
 
+.PHONY: test-e2e-lifecyclehooks
+lifecyclehooks:
+	hack/ci-integration.sh $(GINKGO_ARGS) -focus="Lifecycle" || (hack/junitmerge.sh && exit 1)
```

Comment:
this command is useful to run the test suite separately without having to comment code.